src/mlpack/tests/CMakeLists.txt

```diff
@@ -135,6 +135,7 @@ add_executable(mlpack_test
   main_tests/decision_tree_test.cpp
   main_tests/decision_stump_test.cpp
   main_tests/linear_regression_test.cpp
+  main_tests/local_coordinate_test.cpp
```

Comment:
Please note the algorithm is called "local coordinate coding", not "local coordinate".